pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel", "numpy", "Cython"]
```

Comment:
My understanding is that `wheel` is no longer needed as an explicit dependency and that setuptools should be depending on wheel internally.